src/lib.rs

```diff
@@ -86,6 +86,11 @@ impl Message {
     }
 }
 
+// impl PartialEq for Message {
+//     fn eq(&self, other: &self) -> bool {
+//         self.msg == other.msg;
+//     }
+// }
```

Comment:
generally speaking we do not want to merge commented out code to main

Reply:
Right. Thank you.